src/main/kotlin/org/malachite/estella/commons/models/people/JobSeeker.kt

```diff
@@ -7,5 +7,5 @@ import javax.persistence.*
 data class JobSeeker(
         @Id @Column(name = "id") val id: Int?,
         @OneToOne(cascade = [CascadeType.ALL]) @MapsId @JoinColumn(name = "id") val user: User,
-        @OneToMany(cascade = [CascadeType.ALL]) @JoinColumn(name="job_seeker_id") val files: Set<JobSeekerFile>
+        @OneToMany(cascade = [CascadeType.ALL],fetch = FetchType.EAGER) @JoinColumn(name="job_seeker_id") val files: Set<JobSeekerFile>
```

Comment:
FetchType chyba pasowałby LAZY, bo eager drze wszystko od razu, a jak będzie kilka dużych plików to po co to ładować zawsze z bazy przy zapytaniach o JobSeekera

Reply:
Z lazy testy nie przechodzą XDD